docs/docs-content/clusters/public-cloud/azure/windows.md

```diff
@@ -0,0 +1,167 @@
+---
+sidebar_label: "Deploy Windows Workloads"
+title: "Deploy Microsoft Windows Workloads on an Azure AKS Cluster"
+description:
+  "How to create and manage Windows Node Pools and deploy a Windows-based application on an Azure AKS Cluster."
+hide_table_of_contents: false
+tags: ["public cloud", "azure"]
+sidebar_position: 35
+---
+
+Palette supports the deployment of Microsoft Windows applications on Azure AKS clusters. For the Windows applications to
+operate, a Windows-based node pool is required. This section guides you on creating a Windows node pool within an
+existing AKS cluster and configuring your Windows application to be deployed to that node pool.
+
+## Prerequisites
+
+The following prerequisites must be met before deploying a Windows workload.
+
+- An AKS cluster created as described in the [Create and Manage Azure AKS Cluster](./aks.md) guide.
+
+- A Linux-based node pool configured as the system node pool as described in the
+  [Create a System Node Pool](./aks#create-a-system-node-pool) section of the
+  [Create and Manage Azure AKS Cluster](./aks.md) guide.
+
+- A Windows node pool configured as described in the [Create a Windows Node Pool](#create-a-windows-node-pool) section.
+
+## Enablement
+
+### Create a Windows Node Pool
+
+Follow the steps below to create a Windows node pool within an existing AKS cluster. Refer to the
+[Node Pools](../../cluster-management/node-pool.md) page for more information about node pool configuration.
+
+:::info
+
+Palette also allows you to add a Windows node pool during the creation of an AKS cluster. Refer to the
+[Create and Manage Azure AKS CLuster - Create and Remove Node Pools](./aks#create-and-remove-node-pools) page to learn
+more.
+
+:::
+
+1. Log in to Palette, navigate to the left **Main Menu** and click on **Clusters**.
+
+2. Select your Azure AKS cluster.
+
+3. Navigate to the **Nodes** tab and click on **New Node Pool**.
+
+4. Provide a name for your node pool. When naming a node pool, it is good practice to include a name that matches the
+   node and operating system (OS) in Azure.
+
+5. If auto-scaling is desired, enable the **Enable Autoscaler** option.
+
+<br />
+
+:::warning
+
+Do not select the **System Node Pool** option. System node pools must be Linux-based, and choosing this option will
+remove the ability to create a Windows node pool.
+
+:::
```

Comment:
Should this be indented outside of the list?

Reply:
Nope. Fixed :D